addon/components/polaris-choice-list.js

```diff
@@ -84,6 +91,16 @@ export default Component.extend({
    */
   allowMultiple: false,
 
+  /**
+   * Display an error message
+   *
+   * @property error
+   * @public
+   * @type {string|component}
```

Comment:
These are from a time when I was young and foolish so I know this isn't your doing, but these types should be capitalised since they're class names